pages/404.js

```diff
@@ -1,7 +1,29 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
 export default function FourZeroFour() {
 	return (
-		<div className='404--wrapper'>
-			<h1 className='404--oops'>OOPS!</h1>
+		<div className='fourzerofour--wrapper'>
+			<h1 className='fourzerofour--title'>OOPS!</h1>
+			<ul className='fourzerofour--ul'>
+				<li>
+					<Link href='/'>Home</Link>
+				</li>
+				<li>
+					<Link href='/'>404</Link>
+				</li>
+			</ul>
+			<div className='fourzerofour--error'>
+				<h2 className='fourzerofour--h2'>404</h2>
+				<h3 className='fourzerofour--h3'>Page not found</h3>
+			</div>
+			<p>
+				It looks like nothing was found at this location. Maybe try a search?
+			</p>
+			<label htmlFor='search' hidden>
```

Comment:
`hidden` will prevent screen readers from reading the label. Google a `sr-only` class.